internal_client.go

```diff
@@ -2259,9 +2259,23 @@ func (c *InternalClient) IndexTranslateDataReader(ctx context.Context, index str
 func (c *InternalClient) FieldTranslateDataReader(ctx context.Context, index, field string) (io.ReadCloser, error) {
 	span, ctx := tracing.StartSpanFromContext(ctx, "InternalClient.FieldTranslateDataReader")
 	defer span.Finish()
+	nodes, err := c.Nodes(ctx)
+	if err != nil {
+		return nil, err
+	}
 
+	var primary *disco.Node
```

Comment:
you could use getPrimaryNode([]*disco.Nodes) function from the line 654

Reply:
🤦